lib/pushnotify.js

```diff
@@ -51,11 +39,48 @@ function init(env, ctx) {
       key = notifyToHash(notify);
     }
 
+    notify.key = key;
+
     if (recentlySent.get(key)) {
       console.info('notify: ' + key + ' has ALREADY been sent');
+
       return;
     }
 
+    recentlySent.set(key, notify, 30);
+
+    if (ctx.pushover) sendPushoverNotifications(notify);
+    if (ctx.maker) sendMakerEvent(notify);
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [Expected '{' and instead saw 'sendMakerEvent'.](https://www.codacy.com/app/jasoncalabrese/cgm-remote-monitor/file/758176619/issues/source?bid=2449627&fileBranchId=2449700#l53)